internal/stack/resources.go

```diff
@@ -250,11 +250,11 @@ func addClientCertsToResources(resourceManager *resource.Manager, certResources
 func semverLessThan(a, b string) (bool, error) {
 	sa, err := semver.NewVersion(a)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("%w: %q", err, a)
```

Comment:
Nit. Should we be more explicit on what we are wrapping here?
```suggestion
		return false, fmt.Errorf("failed to parse version %q: %w", a, err)
```

Reply:
I don't think so. The error returned by `semver.NewVersion` is the text "Invalid Semantic Version" which is meaningful. Placing the error in the context that it gets used, with the current version it would be `Error: booting up the stack failed: creating stack files failed: there were 4 errors: template: docker-compose-stack.yml.tmpl:8:7: executing "docker-compose-stack.yml.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: Invalid Semantic Version: <someversion>, template: elasticsearch.yml.tmpl:22:6: executing "elasticsearch.yml.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: Invalid Semantic Version: <someversion>, template: kibana.yml.tmpl:5:52: executing "kibana.yml.tmpl" at <semverLessThan $version "8.15.0-SNAPSHOT">: error calling semverLessThan: Invalid Semantic Version: <someversion>, template: elastic-agent.env.tmpl:8:11: executing "elastic-agent.env.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: Invalid Semantic Version: <someversion>`while the proposed version would give this `Error: booting up the stack failed: creating stack files failed: there were 4 errors: template: docker-compose-stack.yml.tmpl:8:7: executing "docker-compose-stack.yml.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: failed to parse version "<someversion>: Invalid Semantic Version, template: elasticsearch.yml.tmpl:22:6: executing "elasticsearch.yml.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: failed to parse version "<someversion>: Invalid Semantic Version, template: kibana.yml.tmpl:5:52: executing "kibana.yml.tmpl" at <semverLessThan $version "8.15.0-SNAPSHOT">: error calling semverLessThan: failed to parse version "<someversion>: Invalid Semantic Version, template: elastic-agent.env.tmpl:8:11: executing "elastic-agent.env.tmpl" at <semverLessThan $version "8.0.0">: error calling semverLessThan: failed to parse version "<someversion>: Invalid Semantic Version`. I don't see additional text being helpful, and rather, contributing to the overall oververbosity of ep errors.

Reply:
Ok